etc/change_stream_legacy/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
The code for this is great, and it was also super helpful for reviewing this PR. That said, I don't think we'll want to actually check this in when merging this PR, since I don't believe we'll need to use it again now that all the tests have been converted. It's good to keep around for when submitting these conversions to the specs repo though.

Reply:
Is there any downside to leaving it checked in?  I've found it very useful to have artifacts like this easily present when trying to track down change history.

Reply:
I was thinking it might add some noise to greps through the code base, but that's basically it. It's fine with me to keep it checked in if you think it's worth it.